lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -247,7 +247,7 @@ function dosomething_campaign_preprocess_hot_module(&$vars, &$wrapper) {
   $start_date = $vars['hm_start_date'];
   $end_date = $vars['hm_end_date'];
   $readable_end_date = $vars['hm_readable_end_date'];
-  $time_left = dosomething_campaign_get_hot_time_left($node, $vars);
+  $time_left = dosomething_campaign_get_hot_time_left($vars);
```

Comment:
nope, multiple safety checks are in place

https://github.com/DoSomething/phoenix/pull/5940/files#diff-f3333e41a97facbf8cfed1f49cdee095R634

https://github.com/DoSomething/phoenix/pull/5940/files#diff-ae398bb8864fc3c26317f7ccaef04c0aR925